django/cantusdb_project/main_app/views/chant.py

```diff
@@ -692,32 +722,8 @@ def get_context_data(self, **kwargs) -> dict:
         return context
 
     def get_queryset(self) -> QuerySet:
-        values_for_rendering_template = (
-            "id",
-            "folio",
-            "search_vector",
-            "incipit",
-            "manuscript_full_text_std_spelling",
-            "position",
-            "cantus_id",
-            "mode",
-            "manuscript_full_text",
-            "volpiano",
-            "image_link",
-
-            "source__id",
-            "source__title",
-            "source__siglum",
-            "feast__id",
-            "feast__description",
-            "feast__name",
-            "office__id",
-            "office__description",
-            "office__name",
-            "genre__id",
-            "genre__description",
-            "genre__name",
-        )
+        global CHANT_SEARCH_TEMPLATE_VALUES # in order to fetch only those values necessary
```

Comment:
I don't think you need to use `global` here... It's already in the outer scope.

Reply:
you're quite right - thanks